CSS/dropdown.css

```diff
@@ -0,0 +1,13 @@
+.dropdown {
+    margin: 10px auto;
+}
+
+.dropdown p {
+    display: inline-block;
```

Comment:
I dig the "modularized" approach to setting up your style.   Well done.   Most people starting out do not do that.